### PR TITLE
Initial api with book listing functionality.

### PR DESCRIPTION
**Pull request by theTytoAlba**
Written with Django.

To try:
* Go to this folder in terminal.
* Activate environment by typing `source env/bin/activate`
* `cd` into `bookapi` folder while in (env) mode.
*  `python manage.py runserver`
* Now you can try the listing functionality by going into "http://127.0.0.1:8000/book/"

### CODE REVIEW

**Comment by akinarpacioglu** [comment]
Can you make a readme text to let us know that how to import and run this branch?

**Comment by cagdastombul** [comment]
Can you please put everything in a folder for our repository to be cleaner?

**Comment by theTytoAlba** [comment]
* @akinarpacioglu added readme file, please follow the instructions to try and run.  
* @cagdastombul done.

File: Book API Homework/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Book API
+This folder contains an API and an virtualenv.
+
+To copy this branch and work with the API, please follow the steps below:
+
+* Create a folder to copy the branch.
+* Clone the repository: `git clone https://github.com/bounswe2017group7.git`
```
Comment (melihmutlu) [inline Book API Homework/README.md]: Repo link is broken. 
`git clone https://github.com/bounswe/bounswe2017group7.git`

Comment (theTytoAlba) [reply Book API Homework/README.md]: Fixed.

**Pull request merged**